Sources/PrimerSDK/Classes/User Interface/Buttons/PaymentMethodButtonView.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  PaymentMethodButton.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 1/10/21.
+//
+
+import UIKit
+
+class PaymentMethodButtonView: PrimerView {
+    
+    internal private(set) var surCharge: String?
+    internal private(set) var title: String?
+    internal private(set) var image: UIImage?
+    
+    @IBInspectable var cornerRadius: CGFloat {
+        get {
+            return button.layer.cornerRadius
+        }
+        set {
+            button.layer.cornerRadius = newValue
+        }
+    }
+    
+    @IBInspectable var borderWidth: CGFloat {
+        get {
+            return button.layer.borderWidth
+        }
+        set {
+            button.layer.borderWidth = newValue
+        }
+    }
+    
+    @IBInspectable var borderColor: UIColor? {
```

Comment:
we may need to add the option to handle multiple states: error, selected, unselected

Reply:
Agreed, but this isn't the scope of this PR, I believe it's within the theming scope.